tools/system_libs.py

```diff
@@ -1553,8 +1553,10 @@ def install_headers(src_dir, pattern="*.h", target=None):
     if target:
       dest = os.path.join(dest, target)
       shared.safe_ensure_dirs(dest)
-    for f in glob.glob(os.path.join(src_dir, pattern)):
-      logger.debug(os.path.join(dest, os.path.basename(f)))
+    matches = glob.glob(os.path.join(src_dir, pattern))
+    assert matches
```

Comment:
perhaps add an error message? otherwise the error will be unclear.

Reply:
This is an assertion, for developers only.  No user should ever see this, its for debugging while building/modifying ports.

Reply:
But i'll add a message anyway :)